app/stac_api/migrations/0066_merge_20250319_0934.py

```diff
@@ -0,0 +1,13 @@
+# Generated by Django 5.0.11 on 2025-03-19 09:34
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('stac_api', '0065_collectionasset_is_external'),
+        ('stac_api', '0065_remove_asset_add_del_asset_item_update_interval_trigger_and_more'),
+    ]
```

Comment:
Seems not so correct here, your migration should be the 66 not the 65. So you need to revert the migration, then delete those two files and re-create the migrations so that you have only one file with the correct 66 number, see readme about down migrating.